Presentation.Web/app/shared/selectOrgUnit/select2OrgUnit.directive.ts

```diff
@@ -5,6 +5,15 @@
         "$scope",
         "select2LoadingService",
         ($scope: any, select2LoadingService: Kitos.Services.Select2LoadingService) => {
+            if ($scope.renderUnitOriginIndication === true) {
+                const externalUnits = $scope.options.filter(x => x.optionalObjectContext?.externalOriginUuid !== null);
```

Comment:
think you need to extract that options object as a typed variable so we can see what the expected type is. That will make it easier to understand what's behind the "secret" scope properties